os/kerneltc/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config KERNEL_TC 
```

Comment:
Would you not like to give as menu and submenu options? 
selecting KERNEL_TC menu would give options for all drivers and other modules to select or unselect

Reply:
KERNEL_TC is overall config. 
When engineer adds new module test, we can add dependent menu items.